afteraction-share.html

```diff
@@ -2,6 +2,7 @@
 Custom template for same-page post-AJAX submit share ask.
 {% endcomment %}
 
+{{page}}
```

Comment:
Do we need this guy here or am I missing something?

Reply:
I believe this was in the template I originally cloned when working on this back in March! I've tried removing it from my test page and can't see any problems. I've removed it and made another commit to this branch